app/furniture/marketplace/marketplace_component.html.erb

```diff
@@ -2,15 +2,15 @@
   <%= render AlertComponent.new(scheme: :warning, icon: :exclamation_triangle,
                                 title: t('marketplace.marketplace.config_missing_title')) do %>
     <%= t('marketplace.marketplace.config_missing_explainer_html',
-          edit_link: link_to(t('marketplace.marketplace.edit'), location(:edit)) ) %>
+          edit_link: link_to(t('marketplace.marketplace.edit.link_to'), location(:edit)) ) %>
   <% end %>
 <% end %>
 
 <%= render cart %>
 
 <%- if policy(marketplace).edit? %>
   <div class="my-2 text-right">
-    <%= render ButtonComponent.new(scheme: :secondary, label: t('marketplace.marketplace.edit'),
+    <%= render ButtonComponent.new(scheme: :secondary, label: t('marketplace.marketplace.edit.link_to'),
```

Comment:
Genuine curiosity (since I'm still learning):
What is the reason the `button_to` can't be used here?

Reply:
the `MarketplaceComponent` is distinct from the `Marketplace::ManagementComponent`; and `button_to`  is only defined on the `Marketplace::ManagementComponent` at this time.

I could see a world where we do have the `button_to` available on `Marketplace::Component`, which would allow us to use it everywhere!

Reply:
Coolio! I figured it had something to do with the hierarchy, but I wasn't seeing it. Thank you for the clarification!